MailReporter/Mvc/Controllers/MailReporterController.cs

```diff
@@ -82,5 +86,32 @@ public async Task<IActionResult> Mandrill()
             }
             return new OkObjectResult("Event processed successfully");
         }
+
+        [AllowAnonymous]
+        [HttpGet, HttpPost, HttpHead]
+        public async Task<IActionResult> SendInBlue(JObject payload)
+        {
+            //log.Info($"C# HTTP trigger function processed a request. HTTP Method: {req.Method}");
```

Comment:
you can remove all unnecessary comments in the file

Reply:
Removed the extra commented code and updated the ticket id